ember-intl/addon-test-support/add-translations.ts

```diff
@@ -14,42 +12,49 @@ function pickLastLocale(localeName: string | string[]): string {
 }
 
 /**
- * Invokes the `addTranslations` method of the `intl` service. The first
- * parameter, the `localeName`, is optional and will default to the last
- * currently enabled locale. This means, that if you invoke this helper with
- * just translations, they will be added to the last locale and all other
- * locales will be tried before.
+ * Adds translations, as if the end-developer had somehow added them.
+ *
+ * The first parameter, `localeName`, is optional and defaults to
+ * the current locale (the last enabled locale). So if you invoke the
+ * test helper with just the translations, they will be added to the
+ * last locale and all other locales will be tried before.
  *
  * @function addTranslations
  * @param {string} [localeName]
  * @param {object} translations
  */
-export function addTranslations(translations: Translations): void;
-export function addTranslations(
+export async function addTranslations(
+  translations: Translations,
+): Promise<void>;
+export async function addTranslations(
   localeName: string,
   translations: Translations,
-): void;
-export function addTranslations(
+): Promise<void>;
+export async function addTranslations(
   localeNameOrTranslations: string | Translations,
   translations?: Translations,
-): void {
+): Promise<void> {
   const { owner } = getContext() as TestContext;
 
   assert(
-    'The current test context has no owner. Did you forget to call `setupTest(hooks)`, `setupContext(this)` or some other test helper?',
-    typeof owner === 'object' && typeof owner.lookup === 'function',
```

Comment:
This check is brittle, as it knows too much about Ember's implementation. Checking the existence of `owner` should be enough.